diff --git a/node_modules/@uniswap/sdk/dist/constants.d.ts b/node_modules/@uniswap/sdk/dist/constants.d.ts
index 65a9f9b..951c39c 100644
--- a/node_modules/@uniswap/sdk/dist/constants.d.ts
+++ b/node_modules/@uniswap/sdk/dist/constants.d.ts
@@ -5,7 +5,8 @@ export declare enum ChainId {
     ROPSTEN = 3,
     RINKEBY = 4,
     GÖRLI = 5,
-    KOVAN = 42
+    KOVAN = 42,
+    BAOBAB = 1001,
 }
 export declare enum TradeType {
     EXACT_INPUT = 0,
@@ -16,8 +17,8 @@ export declare enum Rounding {
     ROUND_HALF_UP = 1,
     ROUND_UP = 2
 }
-export declare const FACTORY_ADDRESS = "0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f";
-export declare const INIT_CODE_HASH = "0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f";
+export declare const FACTORY_ADDRESS = "0x1c9ab475736b608c290fe8065cfb0f84461c7608";
+export declare const INIT_CODE_HASH = "0x2a710137d93451853521eea4f7d29ac8c3b69b0054825c667c9061bc85d79616";
 export declare const MINIMUM_LIQUIDITY: JSBI;
 export declare const ZERO: JSBI;
 export declare const ONE: JSBI;
