diff --git a/node_modules/@uniswap/sdk/dist/constants.d.ts b/node_modules/@uniswap/sdk/dist/constants.d.ts
index 65a9f9b..a683aa2 100644
--- a/node_modules/@uniswap/sdk/dist/constants.d.ts
+++ b/node_modules/@uniswap/sdk/dist/constants.d.ts
@@ -4,8 +4,13 @@ export declare enum ChainId {
     MAINNET = 1,
     ROPSTEN = 3,
     RINKEBY = 4,
-    GÖRLI = 5,
-    KOVAN = 42
+    BAOBAB = 1001,
+}
+export declare enum ChainsId {
+    MAINNET = 1,
+    ROPSTEN = 3,
+    RINKEBY = 4,
+    BAOBAB = 1001,
 }
 export declare enum TradeType {
     EXACT_INPUT = 0,
diff --git a/node_modules/@uniswap/sdk/dist/index.d.ts b/node_modules/@uniswap/sdk/dist/index.d.ts
index c974b02..ab513e9 100644
--- a/node_modules/@uniswap/sdk/dist/index.d.ts
+++ b/node_modules/@uniswap/sdk/dist/index.d.ts
@@ -1,6 +1,6 @@
 import JSBI from 'jsbi';
 export { JSBI };
-export { BigintIsh, ChainId, TradeType, Rounding, FACTORY_ADDRESS, INIT_CODE_HASH, MINIMUM_LIQUIDITY } from './constants';
+export { BigintIsh, ChainId, ChainsId,TradeType, Rounding, FACTORY_ADDRESS, INIT_CODE_HASH, MINIMUM_LIQUIDITY } from './constants';
 export * from './errors';
 export * from './entities';
 export * from './router';
